src/utils/listing.js

```diff
@@ -11,7 +11,8 @@ import origin from '../services/origin'
  */
 export function dappFormDataToOriginListing(formData) {
   // formData.category data format is "schema.<category>.<subCategory>".
-  const [category, subCategory] = formData.category.split('.').slice(1)
+  const subCategory = formData.category
+  const category = formData.category.split('.')[1]
```

Comment:
These are translation strings Dapp uses for categories: https://github.com/OriginProtocol/origin-dapp/blob/master/src/utils/listingSchemaMetadata.js 

Might be enough to lookup the object with the matching `type` property.

Reply:
Shouldn't category be `${splits[0]}.${splits[1]}` ?
So that for example if subcategory is schema.forSale.antiques then category will be schema.forSale